app.js

```diff
@@ -0,0 +1,106 @@
+// Q1 - Write a console.log statement saying "Hello World!" for each language that you know.
+console.log("Merhaba Dunya");  // Turkish
+console.log("Salam Dunya"); // Azerbaijani
+console.log("Witaj świecie"); //Polish
+
+// Q2 - Consider the following code: console.log('I'm awesome');
+console.log("I'm awesome");
+
+// Q3 - Declare a variable age and initialize it with an integer, using these exact steps:
+let myAge;  // i
+console.log("the value of age will be: 32"); // ii
```

Comment:
Maybe the instructions were not clear. This step was asking you what you thought the value of age was from line 10, _before_ you initialized it.